### DISCUSSION TITLE
How to exclude some components from build?

### DISCUSSION THREAD

**Discussion by wsw70** [quasarframework/quasar#11366] [General - Components / Directives / etc]
I am refactoring my app, component after component. Some components are not compatible with the new architecture and I would like to temporarily exclude them from the build.

**Is there a simple way to do that?**

Notes:
- maybe something similar to Go's `// +build ...` directive?
- the obvious solution I will use for now is to move them somewhere else

**Comment by hawkeye64** [comment]
@wsw70 They should only be included in the build if referenced by an `import`

**Reply by wsw70** [reply]
This is what I thought too, but even when not imported, they are compiled and the errors/warnings they generate are output.

They may not be included in the final bundle (I would say that probably not, as they are not imported) but seem to be at least checked during the build

**Reply by hawkeye64** [reply]
ESLint is being called, you can probably exclude the ones you want by adding them to the `exclude` option of `ESLintPlugin` inside `chainWebpack`.

**Reply by hawkeye64** [reply]
Or, rename the extension of the files to not be `js` or `vue`
